v2/cf-nodejs-logging-support/src/lib/logger/source-utils.ts

```diff
@@ -27,37 +33,90 @@ export class SourceUtils {
         return SourceUtils.instance;
     }
 
-    getFieldValue(source: Source, record: any, writtenAt: Date): string | undefined {
+    isCacheable(source: Source | Source[]): boolean {
+        if (!Array.isArray(source)) {
+            if (["static", "env"].includes(source.type)) {
+                return true;
+            }
+        } else {
+            for (const object of source) {
+                if (["static"].includes(object.type)) {
```

Comment:
```suggestion
                if (object.type == "static") {
```
Alternatively, a switch case would also work here, you decide.

Reply:
I will go with the switch case since it seems more clean 👍